verifier/src/main/res/layout/fragment_home.xml

```diff
@@ -53,11 +53,24 @@
 					android:text="@string/verifier_homescreen_title"
 					android:textSize="@dimen/spacing_very_large" />
 
+				<Button
+					android:id="@+id/covid_certificate_info"
+					style="@style/CovidCertificate.Button.Borderless"
+					android:layout_width="match_parent"
+					android:layout_height="wrap_content"
+					android:layout_gravity="center"
+					android:layout_marginTop="@dimen/spacing_very_large"
+					android:drawableTint="@color/blue"
```

Comment:
Material component button icon should be controlled with `app:iconTint` (https://material.io/components/buttons/android#contained-button)